config/sync/core.entity_form_display.node.recipe.default.yml

```diff
@@ -0,0 +1,175 @@
+uuid: 9b651a4e-7c07-40c6-8ac1-c44f966e68f5
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.node.recipe.field_r_author_quote
+    - field.field.node.recipe.field_r_body
+    - field.field.node.recipe.field_r_categories
+    - field.field.node.recipe.field_r_content_author
+    - field.field.node.recipe.field_r_cooking_time
+    - field.field.node.recipe.field_r_cost
+    - field.field.node.recipe.field_r_featured
```

Comment:
There are issues with this field and the highlighted one. Edit recipe content type and you'll see.

Reply:
done